connaisseur/tests/data/sample_config.yaml

```diff
@@ -0,0 +1,47 @@
+- name: default
```

Comment:
Would maybe also make an explicit test with a config that does not set the `default`s